--- espeak-ng-1.51.1-lbrpdx/Makefile.am	2022-06-21 07:53:53.000000000 -0700
+++ espeak-ng-1.51.1/Makefile.am	2025-11-25 16:35:18.392346887 -0800
@@ -458,7 +458,7 @@
 	phsource/ph_yoruba \
 	phsource/ph_yue \
 	phsource/phonemes \
-	src/espeak-ng
+	src/speak-ng.c
 	ESPEAK_DATA_PATH=$(CURDIR) src/espeak-ng --compile-intonations && \
 		ESPEAK_DATA_PATH=$(CURDIR) src/espeak-ng --compile-phonemes && \
 		touch $@
